src/components/structures/GroupView.js

```diff
@@ -746,13 +746,37 @@ export default React.createClass({
         });
     },
 
+    _leaveGroupWarnings: function() {
+        const warnings = [];
+
+        if (this.state.isUserPrivileged) {
+            warnings.push((
+                <span className="warning">
+                    { " " /* Whitespace, otherwise the sentences get smashed together */ }
+                    { _t("You are an administrator of this community") + ". " }
```

Comment:
we can't safely assume that the full stop will work in other languages - it'd be best to have it as part of the translation

Reply:
also it should be fine to just have a really long string if the concern is the sentences being merged funny

Reply:
Oh right. I was just trying to avoid creating a new string since `You are an administrator of this community` already exists.

Reply:
What is the thing to disable eslint max-len? 

`// eslint-disable-line max-len` or `// eslint-disable-next-line max-len` don't seem to work

Reply:
Oh never mind I just had to do enable and disable 
https://github.com/eslint/eslint/issues/9888#issuecomment-360594658

Reply:
the translation stuff should be fine with something like this:
```
_t(
    "This is supposed " +
    "to be a really long " +
    "line"
);
```

Reply:
Ah ok, should I change to do that? (GitHub is really slow at showing me new comments)

Reply:
if it works, yes please

(github is also really slow for me too)